src/platform_impl/windows/mod.rs

```diff
@@ -19,6 +19,21 @@ pub(self) use crate::platform_impl::Fullscreen;
 
 use crate::event::DeviceId as RootDeviceId;
 use crate::icon::Icon;
+use crate::keyboard::Key;
+
+// ---------------------------------------------
+// WINDOWS MACROS and constants
```

Comment:
there is already a `loword` function a few lines below.
`PRIMARYLANGID` should be added there as well, no need for a new marocs section